worker/index.ts

```diff
@@ -95,7 +95,8 @@ async function processRequest(skinService: MojangRequestService, interpreted: Cr
         case RequestedKind.Cube:
         case RequestedKind.Body: {
             const skin = await skinService.retrieveSkin(interpreted, gatherer);
-            return renderImage(skin, interpreted.size, interpreted.requested, interpreted.armored);
+            const slim = skin.headers.get('X-Crafthead-Skin-Model') === 'slim'
+            return renderImage(skin, interpreted.size, interpreted.requested, interpreted.armored, slim);
```

Comment:
Same here - at his point we're probably better off using
```typescript
            return renderImage(skin, interpreted);
```

and just unpacking in `renderImage`:
```typescript
async function renderImage(skin: Response, request: CraftheadRequest): Promise<Response> {
    const {size, requested, armored} = request;
    // ...
}
```

Reply:
Agreed, I also moved the `slim` initialization into `renderImage` for consistency and to reduce the signature.